<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setup_ip</class>
 <widget class="QWidget" name="setup_ip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color: rgb(255, 255, 255);
}
</string>
   </property>
   <widget class="QPushButton" name="btn_ip_add">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ip_select">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ip_del">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Del</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ip_typing">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="ip_list">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>151</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>1</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <column/>
   </widget>
   <widget class="QLineEdit" name="tp_typing">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="sp_typing">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>90</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>IP ADDRESS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TOUCH PORT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>60</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>SETUP PORT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_tp_add">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>T.P Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_tp_select">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>170</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>T.P Select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_tp_del">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>T.P Del</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sp_del">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>120</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>S.P Del</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sp_select">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>170</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>S.P Select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sp_add">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>90</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>S.P Add</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tp_list">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>111</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>1</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column/>
   </widget>
   <widget class="QTableWidget" name="sp_list">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>150</y>
      <width>111</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>1</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
